spec/directory_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'directory'
+
+describe Directory do
+  it "creates itself successfully without input" do
+    expect(Directory.new).to be_a_kind_of(Directory)
+  end
```

Comment:
I don't think this needs testing. Sure, when you start creating a classic and you've learning and what to be good and strict with your TDD, create the test. But by the point you have behaviour in your class, this test can be deleted. Never be afraid to delete tests that are no longer relevant.